kolibri/utils/options.py

```diff
@@ -49,6 +53,70 @@ def calculate_thread_pool():
     return MIN_POOL
 
 
+ALL_LANGUAGES = "all"
+SUPPORTED_LANGUAGES = "supported"
+BETA_LANGUAGES = "beta"
+
+
+def _set_en_first(value):
+    if "en" in value:
+        value.insert(0, value.pop(value.index("en")))
+
+
+def _coerce_language_value(value):
+    value = str(value)
```

Comment:
coerce from what to what?

Reply:
Pulled this out to avoid complexity complaints from linting, will add some comments.